lib/oci_utils/impl/virt/virt_check.py

```diff
@@ -27,31 +27,23 @@
 _logger = logging.getLogger('oci-utils.virt.virt-check')
 
 
-def iommu_check(check_for_bm_shape=True):
+def iommu_check():
     """
     Verify if the IOMMU is enabled. Linux kernel logs the message "IOMMU:
-    enabled" at boot time and the IOMMU subsystem creates 'dmar' device
-    nodes under /sys/class/iommu # directory. This functions checks for both
+    enabled" at boot time. This functions checks for both
```

Comment:
check for both? I only see one check...

Reply:
sorry . missed on cleanup in the comment. will fix this